src/SDK/KindeProvider.ts

```
@@ -0,0 +1,78 @@
+                body: formData
+            });
+
+            const dataResponse = await response.json();
+            if (dataResponse.error) {
+                reject(dataResponse);
+                return;
+            }
+
+            await Storage.setToken(dataResponse);
+            resolve(dataResponse);
+        });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)